app/services/refresh_reporting_views.rb

```diff
@@ -15,6 +15,7 @@ def self.call
   end
 
   attr_reader :logger
+  delegate :tz, :set_last_updated_at, to: self
```

Comment:
Never seen a delegation to self before; what does this do?

Reply:
It delegates an instance method to the class - `self` is the class when you call `delegate` ->

```ruby
[7] pry(main)> class Test
[7] pry(main)*   puts self
[7] pry(main)*   def self.hi
[7] pry(main)*     puts "hello from a class method"
[7] pry(main)*   end
[7] pry(main)*   delegate :hi, to: self
[7] pry(main)* end
Test
=> [:hi]
[8] pry(main)> Test.new.hi
hello from a class method
```